### ISSUE TITLE
Purchase email header tweaking

### ISSUE THREAD

**Issue by matzipan** [elementary/houston#423]
Hello, 

I just purchased Bookworm through AppCenter and the email confirmation is a bit broken. 

It appears the congrats message uses the application package name instead of the name itself. 

Also, not entirely sure if it's my email client's fault or not, but there appears to be an image at the top which does not get loaded. I don't think it's my client, since the images at the bottom of the email load fine.

![screenshot from 2017-04-28 22 58 43](https://cloud.githubusercontent.com/assets/317239/25548820/3f010e7c-2c66-11e7-844b-4fc683bda9fd.png)

**Comment by lewisgoddard** [comment]
Can you get the URL of the broken image in some way? Right-click maybe?

**Comment by matzipan** [comment]
This is the full URL: https://ci4.googleusercontent.com/proxy/83UaFNZS9fpC2329TR6jrRlloyR3JEDMzowzPTb5XDho_Q1UZzBvbp4u4Xi0nRMYjm8eUxjshrJPTkW0K4cH5K8nZOCHnXrRNJYPcOHcbg_1LDZbXIuppw=s0-d-e1-ft#https://developer.elementary.io/images/system-software-install.svg

I  guess you're only interested in the last part https://developer.elementary.io/images/system-software-install.svg

**Comment by btkostner** [comment]
Interesting. Out of curiosity, what client are you using? When I deployed this change the image worked for me on mail.

**Comment by matzipan** [comment]
Inbox by Gmail. Gmail doesn't work either.

**Comment by lewisgoddard** [comment]
GMail is pretty key, maybe we need to do some research into how they handle images. Maybe SVG is the problem, are the others PNG or JPG?

**Comment by btkostner** [comment]
Yep, just checked all the other images are png. So that is probably the problem. We are going to need to convert the svg icons to png and store them on the repository, or web server...

**Comment by danrabbit** [comment]
Probably the easier thing to do would be just to use the AppCenter icon all the time and store that one icon as a PNG somewhere :p

**Comment by btkostner** [comment]
Ok. I won't disagree :P

**Comment by lewisgoddard** [comment]
I guess that makes more sense, the message is from AppCenter.

**Comment by btkostner** [comment]
I changed the link. Header should be fixed on next deploy.

**Comment by matzipan** [comment]
The issue is also about the package name used instead of the package title. To my mind, it should read "Congrats on installing Bookworm". Should I open another issue?

**Comment by btkostner** [comment]
It's on my list of things to do, but we store the actual project name at all, so there is stuff to do before we can get that working.
